optout/file_handler.go

```diff
@@ -0,0 +1,11 @@
+package optout
```

Comment:
should we add a comment either to the top of this file or underneath the `OptOutFileHandler` that gives a quick overview of the interface + potential for other handlers?